stories/About/Developer Guide.mdx

```diff
@@ -5,6 +5,7 @@ import '../styles.css';
 import client from '../assets/client_component.png';
 
 ## Developer Guide
+<mark>**Note**: Reference of Supabase have been removed as per Shashi. This document must be updated to reference Appwrite instead.</mark>
```

Comment:
Remove this. No need to make a comment about something being completely removed.

Reply:
Done. Committed and will be pushed when all other changes are complete.